template-component/package.json

```diff
@@ -19,23 +19,18 @@
     "dashboard": "cd example && npx convex dashboard",
     "dev": "run-p 'example' 'build:watch'",
     "setup": "npm i && npm run build && cd example && npm i && npx convex dev --once",
-    "build": "npm run build:esm && npm run build:cjs",
     "build:watch": "cd src && npx chokidar '../tsconfig.json' '*.ts' 'react/**/*.ts' 'component/**/*.ts' 'client/**/*.ts' -c 'npm run build' --initial",
-    "build:esm": "tsc --project ./esm.json && npm run copy:dts:esm && echo '{\\n  \"type\": \"module\"\\n}' > dist/esm/package.json",
-    "build:cjs": "tsc --project ./commonjs.json && npm run copy:dts:cjs && echo '{\\n  \"type\": \"commonjs\"\\n}' > dist/commonjs/package.json",
-    "copy:dts:esm": "cpy 'src/**/*.d.ts' 'dist/esm/' --parents",
-    "copy:dts:cjs": "cpy 'src/**/*.d.ts' 'dist/commonjs/' --parents",
+    "build": "tsc --project ./tsconfig.build.json && echo '{\\n  \"type\": \"module\"\\n}' > dist/package.json && npm run copy:dts",
+    "copy:dts": "cpy 'src/**/*.d.ts' 'dist/' --parents",
     "typecheck": "tsc --noEmit",
-    "prepack": "node node10stubs.mjs",
-    "postpack": "node node10stubs.mjs --cleanup",
     "alpha": "rm -rf dist && npm run build && run-p test lint typecheck attw && npm version prerelease --preid alpha && npm publish --tag alpha && git push --tags",
     "release": "rm -rf dist && npm run build && run-p test lint typecheck attw && npm version patch && npm publish && git push --tags",
-    "test": "vitest run --typecheck --config ./src/vitest.config.ts",
-    "test:watch": "vitest --typecheck --config ./src/vitest.config.ts",
-    "test:debug": "vitest --inspect-brk --no-file-parallelism --config ./src/vitest.config.ts",
+    "test": "vitest run --typecheck",
+    "test:watch": "vitest --typecheck",
+    "test:debug": "vitest --inspect-brk --no-file-parallelism",
     "test:coverage": "vitest run --coverage --coverage.reporter=text",
     "lint": "eslint src",
-    "attw": "attw $(npm pack -s) --exclude-entrypoints ./convex.config",
+    "attw": "attw $(npm pack -s) --exclude-entrypoints ./convex.config --profile esm-only",
     "version": "pbcopy <<<$npm_package_version; vim CHANGELOG.md && git add CHANGELOG.md"
   },
   "files": [
```

Comment:
not relevant to the change but files look odd, "react" shouldn't be here since it's inside src/ (my fault)

Reply:
ohhh